src/Compilers/CSharp/Portable/Binder/LockOrUsingBinder.cs

```diff
@@ -80,15 +80,15 @@ protected BoundExpression BindTargetExpression(BindingDiagnosticBag diagnostics,
             if (_lazyExpressionAndDiagnostics == null)
             {
                 // Filter out method group in conversion.
-                var expressionDiagnostics = BindingDiagnosticBag.GetInstance(diagnostics);
```

Comment:
Is the `diagnostics` parameter used anymore in this method?

Reply:
> Is the diagnostics parameter used anymore in this method? 

Yes, eventually ```expressionDiagnostics``` is merged into it.

---
In reply to: [574974823](https://github.com/dotnet/roslyn/pull/51188#discussion_r574974823) [](ancestors = 574974823)

Reply:
Doh, GitHub hid the end of the method and I missed it 😦